impl/client/src/main/javascript/web/pentaho/visual/models/Boxplot.js

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright 2023 Hitachi Vantara. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "pentaho/module!_",
+  "./CategoricalContinuousAbstract",
+  "pentaho/i18n!./i18n/model",
+  "./types/LabelsOption"
+], function(module, BaseModel, bundle, LabelsOption) {
+
+  "use strict";
+
+  return BaseModel.extend({
+    $type: {
+      id: module.id,
+      category: "misc2",
+
+      props: [
+        // VISUAL_ROLES
+        {
+          name: "rows",
+          modes: {dataType: "list"},
+          fields: {isRequired: true}
+        },
+        {
+          name: "lowerQuartile",
+          base: "pentaho/visual/role/Property",
+          modes: {dataType: "number"},
+          ordinal: 11
+        },
```

Comment:
I believe this chart requires at least one measure VR to be mapped.
To implement this rule, use the same technique as used by, e.g., https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/6479eecfa7f67e3152ebaa73a639c01ac90ef3a4/impl/client/src/main/javascript/web/pentaho/visual/models/BarLine.js#L38-L50.